### PR TITLE
MODINREACH-456 Updating pom.xml and interface dependencies for Ramsons

### PR DESCRIPTION
**Pull request by Vignesh-kalyanasundaram**
#### TODOS and Open Questions

- [ ] Check logging

## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -22,29 +22,29 @@
 
   <properties>
     <!-- runtime dependencies -->
-    <folio-service-tools.version>4.0.0</folio-service-tools.version>
+    <folio-service-tools.version>4.0.1</folio-service-tools.version>
```
Comment (julianladisch) [inline pom.xml]: Can we use the Ramsons version 4.1.0? https://github.com/folio-org/folio-service-tools/releases/tag/v4.1.0

File: pom.xml [hunk 2]
```diff
@@ -22,29 +22,29 @@
...
     <folio-spring.version>8.2.0</folio-spring.version>
     <spring-cloud-starter-openfeign.version>4.1.3</spring-cloud-starter-openfeign.version>
     <commons-collections4.version>4.4</commons-collections4.version>
-    <mapstruct.version>1.5.3.Final</mapstruct.version>
+    <mapstruct.version>1.6.2</mapstruct.version>
     <marc4j.version>2.9.2</marc4j.version>
```
Comment (julianladisch) [inline pom.xml]: Can we use the latest version? https://github.com/marc4j/marc4j/releases/tag/2.9.6

File: pom.xml [hunk 3]
```diff
@@ -22,29 +22,29 @@
...
     <coffee-boots.version>3.0.0</coffee-boots.version>
```
Comment (julianladisch) [inline pom.xml]: Can we use latest version 4.0.0?

File: pom.xml (hunk 1 above)
Comment (Vignesh-kalyanasundaram) [reply pom.xml]: Hi, updated folio-service-tools to 4.1.0

File: pom.xml (hunk 2 above)
Comment (Vignesh-kalyanasundaram) [reply pom.xml]: I updated the marc4j version as mentioned

File: pom.xml (hunk 3 above)
Comment (Vignesh-kalyanasundaram) [reply pom.xml]: I updated the coffee boots version as mentioned

**Review by julianladisch — APPROVED** [review]
Looks good, thanks!

**Review by gurleenkaurbp — APPROVED** [review]

**Pull request merged**